analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.cs

```diff
@@ -908,3 +1001,58 @@ struct StructWithValuePropertyAndCastOperators
     }
 }
 
+namespace TypeWithValueStaticProperty
+{
+    class Test
+    {
+        void Basics()
+        {
+            _ = AClassWithStaticValueProperty.Value;                                // Compliant, not on nullable value type
+            _ = AClassWithStaticValueProperty.Value.Value;                          // Compliant
+            _ = AClassWithStaticValueProperty.Value.Value.InstanceProperty;         // Compliant
+            _ = AClassWithStaticValueProperty.Value.Value.InstanceProperty.Value;   // Compliant
+            _ = new AClassWithInstanceValueProperty().Value;                        // Compliant
+        }
+    }
+
+    class AClassWithStaticValueProperty
+    {
+        public AClassWithInstanceValueProperty InstanceProperty => new AClassWithInstanceValueProperty();
+
+        public static AClassWithInstanceValueProperty Value => new AClassWithInstanceValueProperty();
+    }
+
+    class AClassWithInstanceValueProperty
+    {
+        public AClassWithStaticValueProperty Value => new AClassWithStaticValueProperty();
```

Comment:
Two changes:
* Make it an auto property because of #7007
* Add a Test with access to `this.Value` in an instance method of this class.

Reply:
I think [this](https://github.com/SonarSource/sonar-dotnet/pull/7006#discussion_r1154638648) comment applies here too.